### ISSUE TITLE
Allow ignoring non existent files by an option.

### ISSUE THREAD

**Issue by jjangga0214** [eslint/eslint#12959]
**The version of ESLint you are using.**
6.7.2

**The problem you want to solve.**
`eslint path/to/non-existent-file.js` throws an error like this.

```bash
No files matching 'path/to/non-existent-file.js' were found.
```

**Your take on the correct solution to problem.**
Allow ignoring non existent files by an option.

**Are you willing to submit a pull request to implement this change?**
If available.

**Comment by aladdin-add** [comment]
do you mean something like [no-error-on-unmatched-pattern](https://eslint.org/docs/user-guide/command-line-interface#no-error-on-unmatched-pattern) ?

**Comment by jjangga0214** [comment]
@aladdin-add 
Oh, Thank you so much to let me know it!
It works as expected.
Thanks!
